<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplesWidget</class>
 <widget class="QWidget" name="SamplesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SamplesWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelHeading">
     <property name="text">
      <string>Heading:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditHeading"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelDOI">
     <property name="text">
      <string>DOI:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditDOI"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPath">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditPath"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonPathFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelAnnotation">
     <property name="text">
      <string>Annotation:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditAnnotation"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelView">
     <property name="text">
      <string>View:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2">
    <widget class="QComboBox" name="comboBoxView">
     <item>
      <property name="text">
       <string>--</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" rowspan="2">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEditDescription"/>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
